app/helpers/comments_helper.rb

```diff
@@ -0,0 +1,5 @@
+module CommentsHelper
+  def can_manage_comment?(comment)
+    comment.user == current_user
```

Comment:
instead of helper use something like

current_user.can_manage?(comment) for now

Reply:
added method to User model